notebooks/openeo_vito_example.ipynb

```diff
@@ -17,9 +26,17 @@
     "import matplotlib.pyplot as plt"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0c2bfc66",
+   "metadata": {},
+   "source": [
+    "Connect to the openEO backened (2) to create a test polygon (3) and a datacube (4)."
```

Comment:
Typo: backend

We don't really need this to create the test polygon.

Reply:
Fixed (and polygon part rephrased)

Reply:
It's "backned" now 😄 

fftm after fixing

Reply:
🤦‍♂️